cmd/server/server.go

```diff
@@ -12,20 +12,20 @@ var s *service.Service
 
 func StartServer() {
 	options := config.DefaultOptions
-	//str, err := structure.NewStringStructure(options)
-	//if err != nil {
-	//	fmt.Println("flydb start error: ", err)
-	//	return
-	//}
-	//s = service.NewService(config.DefaultAddr, str)
-	//s.StartServer()
-	hash, err := structure.NewHashStructure(options)
+	str, err := structure.NewStringStructure(options)
```

Comment:
You must pull the latest code from the main branch and resolve conflicts before you can commit pr.
![image](https://github.com/ByteStorage/FlyDB/assets/90094807/71edbecd-ab70-4cd0-9778-fb0cddadca8a)

Reply:
> You must pull the latest code from the main branch and resolve conflicts before you can commit pr. ![image](https://user-images.githubusercontent.com/90094807/253785904-71edbecd-ab70-4cd0-9778-fb0cddadca8a.png)

i am sorry.it is my mistake. Now these files are ok. thanks a lot.